scripts/tests/api_compare/docker-compose.yml

```diff
@@ -60,6 +61,43 @@ services:
       timeout: 10m
       retries: 3
       start_period: 10m
+  forest-rpc-ready:
+    build:
+      context: ../../../.
+      dockerfile: ${FOREST_DOCKERFILE_OVERRIDE:-Dockerfile}
+    depends_on:
+      forest:
+        condition: service_healthy
+    networks:
+      - api-tests
+    entrypoint: [ "/bin/bash", "-c" ]
+    user: 0:0
+    command:
+      - |
+        set -uo pipefail
+        type curl >/dev/null 2>&1 || {
+          echo >&2 "curl is required but it's not installed. installing.";
+          apt-get update && apt-get install -y curl
```

Comment:
This means `curl` needs to be installed on every `compose up`. It'd be nicer to embed it in an image.

Reply:
Using now some healthcheck subcommand so we don't prereq `curl`.